isaric/parsers/isaric_washington.toml

```diff
@@ -0,0 +1,614 @@
+]
+
+
+[visit.treatment_inhaled_nitric_oxide]
+field = "treatment_events___2"
+ref = "Y/N/NK"
+
+[visit.treatment_noninvasive_ventilation]
+combinedType = "any"
+fields = [
+    { field = "oxygenation___3", ref = "Y/N/NK" },
+    { field = "oxygenation_methods_2___2", ref = "Y/N/NK" },
+]
+
+[visit.treatment_high_flow_nasal_cannula]
+combinedType = "any"
+fields = [
+    { field = "oxygenation___1", ref = "Y/N/NK" },
+    { field = "oxygenation_methods_2___3", ref = "Y/N/NK" },
+]
+[visit.treatment_steroids]
+field = "medications___16" 
+ref = "Y/N/NK"
+
+[visit.treatment_intravenous_fluids]
+field = "totak_iv_fluid"
+
+[visit.treatment_nsaid]
+field = "medications___16" 
+ref = "Y/N/NK"
+
+[visit.treatment_neuraminidase]
+field = "name_of_antiviral___4" 
+ref = "Y/N/NK"
+
+[visit.treatment_neuromuscular_blocking_agents]
+field = "nmb_given"
+description = "NeuroMuscular Blockers (NMB) Given?"
+values = { 0 = false, 1 = true, 2 = true, 3 = true, 4 = true, 5 = true, 6 = true, 7 = true }
+
+[visit.treatment_cpr] # not sure about this!
+field = "good_death___1"
+ref = "Y/N/NK"
+
+[visit.treatment_tocilizumab]
+field = "mis_medications___1"
+ref = "Y/N/NK"
+
+[visit.treatment_immunoglobulins]
+field = "blood_products___6"
+ref = "Y/N/NK"
+
+[visit.outcome]
+field = "hosp_status"
+description = "Discharged alive or dead"
+values = { 1 = "Deceased", 0 = "Alive" }
+
+[visit.date_outcome] # calculate: hospital length of stay (hospital_los) + the admission date - CHECK THIS!
+field = "admission_date"
+description = "outcome date calculated from admission and hospital los"
+apply = { function = "endDate", params = ["hospital_los"]}
+
+
+[[observation]] 
+name = "avpu"
+date = "date_filling_first_form"
+phase = "study"
+text = { field = "avpu_score_v2___1", values = { 1 = "Alert" } }
+
+[[observation]] 
+name = "avpu"
+date = "date_filling_first_form"
+phase = "study"
+text = { field = "avpu_score_v2___2", values = { 1 = "Verbal" } }
+
+[[observation]] 
+name = "avpu"
+date = "date_filling_first_form" 
+phase = "study"
+text = { field = "avpu_score_v2___3", values = { 1 = "Pain" } }
+
+[[observation]] 
+name = "avpu"
+date = "date_filling_first_form" 
+phase = "study"
+text = { field = "avpu_score_v2___4", values = { 1 = "Unresponsive" } }
+
+[[observation]] 
+name = "diastolic_blood_pressure_mmHg"
+date = "date_filling_first_form"
+phase = "study"
+value = { field = "dbp_echo" }
+
+[[observation]] # - CHECK THIS!!!
+name = "from_same_blood_gas_record_as_pao2"
+date = "date_filling_first_form"
+phase = "study"
+value = { field = "pco2_from_same_blood_gas_r" }
+
+[[observation]] # - CHECK THIS!!!
+name = "from_same_blood_gas_record_as_pao2"
+date = "date_filling_first_form"
+phase = "study"
+value = { field = "ph_from_same_blood_gas_rec" }
+
+[[observation]] # - CHECK THIS!!!
+name = "from_same_blood_gas_record_as_pao2"
+date = "date_filling_first_form" 
+phase = "study"
+value = { field = "hco3_from_same_blood_gas_r" }
```

Comment:
This is recording a value for hco3, which we don't currently have in the schema

Reply:
Corrected!